azure/scope/managedcontrolplane_test.go

```diff
@@ -272,28 +265,10 @@ func TestManagedControlPlaneScope_PoolVersion(t *testing.T) {
 	}
 
 	for _, c := range cases {
-		c := c
```

Comment:
I guess we can ditch this now with Go 1.22?

Reply:
I think this change was probably safe even before that since these tests aren't running in parallel.